.circleci/config.yml

```diff
@@ -137,17 +129,9 @@ jobs:
           sudo service grafana-server start
           sudo grafana-cli --version
     - run:
-        name: Run e2e tests
+        name: Run tests
         command: |
-          npx grafana-toolkit plugin:ci-test
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ci/jobs/test_6_3_0_beta1
-    - store_test_results:
-        path: ci/jobs/test_6_3_0_beta1
-    - store_artifacts:
-        path: ci/jobs/test_6_3_0_beta1
+          npx grafana-toolkit plugin:test
```

Comment:
Yeah, thought similar things, the CI config needs either minor or major revamping. At the very least, probably reasonably current versions of grafana should be used for testing.
Which... also reminds of the fact that at least one e2e test example would probably be good because I figure - what's the point of running grafana if there is no e2e test?

Reply:
Yea, e2e tests are a good point, too!